package.json

```diff
@@ -9,11 +9,13 @@
     "test": "jest --watch"
   },
   "dependencies": {
+    "react-dom": "16.13.1",
     "next": "9.4.2",
     "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "data-explorer": "git+https://github.com/datopian/data-explorer.git"
   },
   "devDependencies": {
+    "@datopian/datapackage-views-js": "^1.3.15",
```

Comment:
Not sure if you need this, if you're using `data-explorer`

Reply:
Yeah, I dont think I need it. Do you think I should close this PR since it's obsolete now and we're going look into implementing data package preview in a different way?